docs/api.md

```diff
@@ -0,0 +1,263 @@
+## Single Session API
+
+### login(options) => Promise:void
+Kick off the login process for the user:
+
+```typescript
+import { login } from '@inrupt/solid-client-authn-browser';
```

Comment:
FYI, the current package does not export a `login` function: https://github.com/inrupt/solid-client-authn/blob/master/src/index.ts

Reply:
Fixed in 569e012104773ba0e4075b9e821c24ce327e95f7, along with other issues with the former API reference.